src/commands/apps/create.ts

```diff
@@ -46,6 +46,12 @@ export default class AppCreateCommand extends Command {
   @hasArg
   platform: string;
 
+  @help("The environment of the app. This must be one word, alphanumeric, first letter capitalized.")
```

Comment:
If it's a requirement that the first letter should be capitalized - we should either check that here - or reformat the input and warn the user.

Reply:
Pushed a commit which should address this by adding client-side validation.